planning/index.ftd

```diff
@@ -51,7 +53,8 @@ the scripts etc here.
 - R: using css for things fastn is lacking
 - R: integration with JS
 
-- R: How to use icon libraries like `react-icons` or `font-awesome` or any external icon library in fastn
+- R: How to use icon libraries like `react-icons` or `font-awesome` or any 
+external icon library in fastn
```

Comment:
In the markdown list, when wrapping a line, make it indent by as many spaces as the first (non `-`) character of previous line.